lib/frontend/src/frontend.rs

```diff
@@ -571,10 +571,57 @@ impl<'a> FunctionBuilder<'a> {
         self.ins().call(libc_memcpy, &[dest, src, size]);
     }
 
+    /// Optimised memcpy for small copys.
+    pub fn emit_small_memcpy(&mut self,
+                             isa: &TargetIsa,
+                             dest: Value,
+                             src: Value,
+                             size: u64,
+                             dest_align: u8,
+                             src_align: u8)
+    {
+        const THRESHOLD: u64 = 4;
```

Comment:
Please add a comment mentioning that this size is just a guess, not based on experiments.